Sloth.Core/Domain/Integration.cs

```diff
@@ -38,8 +37,7 @@ public class Integration
         [MaxLength(128)]
         public string HoldingAccount { get; set; }
 
-        public IList<CybersourceBankReconcileJobBlob> CybersourceBankReconcileJobBlobs { get; set; }
-
+        public IList<TransactionBlob> TransactionBlobs { get; set; }
```

Comment:
Not a fan of using the word transaction here. Call it JobBlobs ? JobRecordBlobs?

Reply:
But it's a new many-to-many between Transactions and Blobs, not JobRecords and Blobs.